assembl/static2/js/app/components/common/navigationMenu.jsx

```diff
@@ -2,17 +2,18 @@ import React from 'react';
 import { connect } from 'react-redux';
 import { Translate } from 'react-redux-i18n';
 import { Link } from 'react-router';
+import { Routes } from '../../routes';
 
 class NavigationMenu extends React.Component {
   render() {
     const { debateData } = this.props.debate;
-    const { rootPath } = this.props.context;
+    // TODO: When turnling url into Link, do not forget activeClassName="active"
```

Comment:
Ahh, no no, this comment is for if we ever make a `<Link>` further below. There was an activeClassName on there. This comment is not very relevant at the moment. I should re-phrase it.